standalone-metastore/src/main/java/org/apache/hadoop/hive/metastore/HiveMetaStoreClient.java

```diff
@@ -456,6 +457,7 @@ private void open() throws MetaException {
                   trustStorePath, trustStorePassword );
               LOG.info("Opened an SSL connection to metastore, current connections: " + connCount.incrementAndGet());
             } catch(IOException e) {
+              ioException = e;
```

Comment:
This assignment is excessive cause it's not doing anything after the IllegalArgumentException is thrown on the next line.

Reply:
Thank you @difin for the review.
You're right! 

if the `IllegalArgumentException` isn't explicitly intended to terminate reattempts, I can change it to `MetaException` to log the error message and allow the connection process to make another attempt.

If not, I can remove the assignment.

Reply:
@deniskuzZ, could you please review the options mentioned above and let us know which one you think we should proceed with?

Reply:
Hive-3.x should be declared. EOL soon